Custom Device Source/Docs/Modules.md

```diff
@@ -0,0 +1,87 @@
+
+## Analog Input
+| Model | ADC Resolution | Description | Channels | Measurement Type | Units |
+|:---:|:---:|---|:---:|:---:|:---:|
+| 9201 | 12-Bit | Analog Input | 8 | Voltage | V |
+| 9203 | 16-Bit | Analog Input | 8 | Current | A |
+| 9205 | 16-Bit | Analog Input | 32 | Voltage | V |
+| 9206 | 16-Bit | Analog Input for Fuel Cells | 32 | Voltage | V |
+| 9207 | 24-Bit | Isolated Analog Input | 8V + 8A | Voltage + Current | V , A |
+| 9208 | 24-Bit | Isolated Analog Input | 16 | Current | A |
+| 9209 | 24-Bit | 60 Vdc Isolated Analog Input | 16 | Voltage | V |
+| 9210 | 24-Bit | Thermocouple Input | 4 | Temperature or Voltage | Custom |
+| 9211 | 24-Bit | Thermocouple Input | 4 | Temperature or Voltage | Custom |
+| 9212 | 24-Bit | 250 Vrms Ch-Ch Isolation Thermocouple Input | 8 | Temperature or Voltage | Custom |
+| 9213 | 24-Bit | Thermocouple Input | 16 | Temperature or Voltage | Custom |
+| 9214 | 24-Bit | Thermocouple Input | 16 | Temperature or Voltage | Custom |
+| 9215 | 16-Bit | Simultaneous Analog Input | 4 | Voltage | V |
+| 9216 | 24-Bit | RTD Input | 8 | Temperature or Resistance | Custom |
+| 9217 | 24-Bit | RTD Input | 4 | Temperature or Resistance | Custom |
+| 9219 | 24-Bit | Universal Analog Input | 4 | Universal* | Custom |
+| 9220 | 16-Bit | Analog Input | 8 | Voltage | V |
+| 9221 | 12-Bit | Analog Input | 8 | Voltage | V |
+| 9222 | 16-Bit | Analog Input | 4 | Voltage | V |
+| 9223 | 16-Bit | Analog Input | 4 | Voltage | V |
+| 9224 | 24-Bit | Analog Input | 8 | Voltage | V |
+| 9225 | 24-Bit | Simultaneous Analog Input | 3 | Voltage | V |
+| 9226 | 24-Bit | RTD Input | 8 | Temperature or Resistance | Custom |
+| 9227 | 24-Bit | Simultaneous Analog Input | 4 | Current | A |
+| 9228 | 24-Bit | Analog Input | 8 | Voltage | V |
+| 9229 | 24-Bit | Simultaneous Analog Input | 4 | Voltage | V |
+| 9233 | 24-Bit | IEPE Analog Input | 4 | Voltage | V |
+| 9234 | 24-Bit | SW Selectable IEPE & AC/DC | 4 | Voltage | V |
+| 9235 | 24-Bit | Quarter Bridge Analog Input | 8 | Strain | V/V |
+| 9236 | 24-Bit | Quarter Bridge Analog Input | 8 | Strain | V/V |
+| 9237 | 24-Bit | Half/Full Bridge Analog Input | 4 | Strain | V/V |
+| 9239 | 24-Bit | Simultaneous Analog Input | 4 | Voltage | V |
+| 9242 | 24-Bit | 3-Phase + Neutral 400 Vrms L-L | 4 | Current | A |
+| 9244 | 24-Bit | 3-Phase + Neutral 800 Vrms L-L | 4 | Current | A |
+| 9253 | 24-Bit | Analog Input | 8 | Current | A |
+
+## Analog Output
+| Model | DAC Resolution | Channels | Range | Channel Type | Units |
+|:---:|---|:---:|:---:|:---:|:---:|
+| 9263 | 16-Bit | 4 | +/- 10 V | Voltage | V |
+| 9264 | 16-Bit | 16 | +/- 10 V | Voltage | V |
+| 9265 | 16-bit | 4 | 0-20 mA | Current | A |
+| 9266 | 16-Bit | 8 | +/- 20 mA | Isolated Current | A |
+| 9269 | 16-Bit | 4 | +/- 10 V | Isolated Voltage | V |
+
+***
+
+## Digital Modules
+| Model | Description | Lines and Direction |
+|:---:|---|:---:|
+| 9344 | User Interface with Switch + LED | 4 DO |
```

Comment:
Not checking every module, but I happen to know this one is technically 4 DI and 4 DO, because the switches are inputs and the LEDS are addressable independently.

Reply:
```suggestion
| 9344 | User Interface with Switches + LEDs | 4 DI + 4 LED |
```